src/constants/constants.jsx

```diff
@@ -1,91 +1,100 @@
-export const BALANCES_LIGHT_RETURNED = 'BALANCES_LIGHT_RETURNED'
-
-export const GET_VAULT_BALANCES = 'GET_VAULT_BALANCES'
-export const VAULT_BALANCES_RETURNED = 'VAULT_BALANCES_RETURNED'
+export const GET_BALANCES_LIGHT = "GET_BALANCES_LIGHT";
+export const BALANCES_LIGHT_RETURNED = "BALANCES_LIGHT_RETURNED";
 
+export const GET_VAULT_BALANCES = "GET_VAULT_BALANCES";
+export const VAULT_BALANCES_RETURNED = "VAULT_BALANCES_RETURNED";
 
 // #Dashboard
-export const GET_DASHBOARD_SNAPSHOT = 'GET_DASHBOARD_SNAPSHOT'
-export const DASHBOARD_SNAPSHOT_RETURNED = 'DASHBOARD_SNAPSHOT_RETURNED'
-
-export const GET_USD_PRICE = 'GET_USD_PRICE'
-export const USD_PRICE_RETURNED = 'USD_PRICE_RETURNED'
+export const GET_DASHBOARD_SNAPSHOT = "GET_DASHBOARD_SNAPSHOT";
+export const DASHBOARD_SNAPSHOT_RETURNED = "DASHBOARD_SNAPSHOT_RETURNED";
 
+export const GET_USD_PRICE = "GET_USD_PRICE";
+export const USD_PRICE_RETURNED = "USD_PRICE_RETURNED";
 
 // #Earn
-export const INVEST = 'INVEST'
-export const INVEST_RETURNED = 'INVEST_RETURNED'
-
-export const INVEST_ALL = 'INVEST_ALL'
-export const INVEST_ALL_RETURNED = 'INVEST_ALL_RETURNED'
+export const INVEST = "INVEST";
+export const INVEST_RETURNED = "INVEST_RETURNED";
 
-export const REDEEM = 'REDEEM'
-export const REDEEM_RETURNED = 'REDEEM_RETURNED'
+export const INVEST_ALL = "INVEST_ALL";
+export const INVEST_ALL_RETURNED = "INVEST_ALL_RETURNED";
 
+export const REDEEM = "REDEEM";
+export const REDEEM_RETURNED = "REDEEM_RETURNED";
 
 // #Vaults
-export const DEPOSIT_VAULT = 'DEPOSIT_VAULT'
-export const DEPOSIT_VAULT_RETURNED = 'DEPOSIT_VAULT_RETURNED'
-
-export const DEPOSIT_ALL_VAULT = 'DEPOSIT_ALL_VAULT'
-export const DEPOSIT_ALL_VAULT_RETURNED = 'DEPOSIT_ALL_VAULT_RETURNED'
-
-export const WITHDRAW_VAULT = 'WITHDRAW_VAULT'
-export const WITHDRAW_VAULT_RETURNED = 'WITHDRAW_VAULT_RETURNED'
-
-export const WITHDRAW_ALL_VAULT = 'WITHDRAW_ALL_VAULT'
-export const WITHDRAW_ALL_VAULT_RETURNED = 'WITHDRAW_ALL_VAULT_RETURNED'
-
-
+export const DEPOSIT_CONTRACT = "DEPOSIT_CONTRACT";
+export const DEPOSIT_CONTRACT_RETURNED = "DEPOSIT_CONTRACT_RETURNED";
+export const DEPOSIT_CONTRACT_RETURNED_COMPLETED =
+  "DEPOSIT_CONTRACT_RETURNED_COMPLETED";
+
+export const DEPOSIT_ALL_CONTRACT = "DEPOSIT_ALL_CONTRACT";
+export const DEPOSIT_ALL_CONTRACT_RETURNED = "DEPOSIT_ALL_CONTRACT_RETURNED";
+export const DEPOSIT_ALL_CONTRACT_RETURNED_COMPLETED =
+  "DEPOSIT_ALL_CONTRACT_RETURNED_COMPLETED";
+
+export const WITHDRAW_VAULT = "WITHDRAW_VAULT";
+export const WITHDRAW_VAULT_RETURNED = "WITHDRAW_VAULT_RETURNED";
+export const WITHDRAW_VAULT_RETURNED_COMPLETED =
+  "WITHDRAW_VAULT_RETURNED_COMPLETED";
+
+export const WITHDRAW_BOTH = "WITHDRAW_BOTH";
+
+export const WITHDRAW_BOTH_VAULT = "WITHDRAW_BOTH_VAULT";
+export const WITHDRAW_BOTH_VAULT_RETURNED = "WITHDRAW_BOTH_VAULT_RETURNED";
+export const WITHDRAW_BOTH_VAULT_RETURNED_COMPLETED =
+  "WITHDRAW_BOTH_VAULT_RETURNED_COMPLETED";
```

Comment:
Added a new "state" xxx_COMPLETED to indicate the successful deposit or withdrawal